_blog/articles/2024-06-24-aws-bookshelf.md

```diff
@@ -89,10 +89,7 @@ permalink: /aws-bookshelf
 ## S3
 
 - [Creating an S3 Bucket and Exploring the UI](https://scribehow.com/shared/Creating_an_S3_Bucket_and_Exploring_the_UI__0TFMdpbkQvS6jOisdkQjeA)
-- [Understanding S3: Objects, Buckets, and Key-Value Pairs](https://scribehow.com/shared/Understanding_S3**Objects_Buckets_and_Key_Value_Pairs**rE1C8p-eQlqBp10AbmpjzQ)
-- [How S3 Versioning Works: Deep Dive and Hands-On](https://scribehow.com/shared/How_S3_Versioning_Works__Deep_Dive_and_Hands_On__uzPEzOlzSaK8_s8MXHdAmg)
-- [Working with S3 Encryption](https://scribehow.com/shared/Working_with_S3_Encryption__MCgNQmKiTfC5SwpZBxmhgQ)
-- [Hosting Static Website in S3](https://scribehow.com/shared/Hosting_Static_Website_in_S3__aI1hLU-4Rt-FRrxZ0QOZWg)
+- [Understanding S3: Objects, Buckets, and Key-Value Pairs](https://scribehow.com/shared/Understanding_S3__Objects_Buckets_and_Key_Value_Pairs__x1GtUtZKS9axATcFzF1hTQ)

```

Comment:
Why did we remove too many links? are they all deleted? i was expecting updates to the URL but not deletion.

Reply:
resolved.